ocaml/idl/datamodel.ml

```diff
@@ -3747,6 +3749,7 @@ let pool_patch_clean = call
   ~doc:"Removes the patch's files from the server"
   ~in_oss_since:None
   ~in_product_since:rel_miami
+  ~internal_deprecated_since:rel_dundee
```

Comment:
Nowadays we should use lifecycle rather than internal_deprecated_since.
That allows for a string describing the reason for deprecation.